sippy-ng/src/constants.js

```diff
@@ -119,6 +119,12 @@ export const BOOKMARKS = {
     operatorValue: '>',
     value: '1',
   },
+  WATCHLIST: {
+    columnField: 'watchlist',
+    not: false,
+    operatorValue: 'equals',
+    value: 'true',
```

Comment:
Looks like this gets treated as a string? Does it work in the UI (I haven't tested it yet)?

Do we use the go-based filtering anywhere anymore? We should probably make sure watchlist is included in https://github.com/openshift/sippy/blob/master/pkg/apis/api/types.go#L435.  There's no bool type, but maybe string  works.

Reply:
I did add it to the Test type in that file, unless I'm missing what you meant just let me know.

String does appear to work for the filtering.

Reply:
It's added to the type but not the go filtering. There's helpers that categorize the field and return the value like https://github.com/openshift/sippy/blob/2c79ecc2739fb34d21477e4390ceb793ec9b0893/pkg/apis/api/types.go#L480.  It's used if filters are processed by go and not sql.

Reply:
Ah now I understand, thanks. Added

Reply:
You're still missing the entry in `GetStringValue()`.  Sorry, this filtering thing is awkward but I think it's still used a few places.

Reply:
Sorry, added that one too.